core/communication/bubblewrap/src/main/javascript/index.ts

```diff
@@ -218,50 +229,20 @@ export class Bubblewrap {
 
     /**
      * Constructs a new, independent [[Bubblewrap]] instance with additional registered classes.
+     * There must be a reason for this creating an additional object; unclear to me right now
      *
      * This method throws an exception if there is a duplicate class identifier.
      */
     addClasses(more: Classes): Bubblewrap {
-        const thisKeys = Object.keys(this.classes);
-        const thatKeys = Object.keys(more);
-        for (const thisKey of thisKeys)
-            if (thatKeys.includes(thisKey)) throw new Error(`Duplicate identifier ${thisKey}`);
+        const theseKeys = Object.keys(this.classes);
+        const thoseKeys = Object.keys(more);
+        for (const thatKey of thoseKeys)
+            if (theseKeys.includes(thatKey)) throw new Error(`Duplicate identifier ${thatKey}`);
```

Comment:
That rename seems a bit pointless to me - `this` and `that` is pretty common naming, `these` and `those` I've never seen. It's actually `this' keys`, it's not `these keys`, because there's only one `this` here, but multiple `keys`.